package.json

```diff
@@ -144,12 +145,12 @@
     "sass-loader": "7.0.3",
     "shelljs": "0.8.2",
     "style-loader": "0.21.0",
-    "ts-loader": "4.3.1",
+    "ts-loader": "4.4.2",
```

Comment:
I think we can remove this dependencies ?

Reply:
see https://github.com/coveo/react-vapor/pull/746#discussion_r206294859 as of now